### ISSUE TITLE
Launch BF placeholder

### ISSUE THREAD

**Issue by considine** [considine/email-crawler#16]
# Launch BF: Getcho

## TL;DR 

Getcho is local delivery platform for high-value, mission-critical goods. The Getcho layer sits on top of the fleet layer -- sort of like how the TCP layer is built on top of UDP.

![](https://mermaid.ink/img/pako:eNp1Us1qwzAMfhXh0wbpC-RQGA10vZSywE65eLbSmvkns-WWUPruc-ImKXTzyUjfn2RfmXASWckC_kS0AivFj56bxkI6HfekhOq4JahU6DiJE_rn3hZTwz3XNzGQM38xKq_OQz13Fu3Vep3FStjjBSrUA66HjyFeoAzPiARdeCW8iW_rLhrlEWfapD8TpkAlfKJXbT8nhJ1ES4r6TJjKD3EmAGycbZU3KJ_U81Al1GjlEn1nA_koSDkb4GV7qIuUVWAIcPCOnHD69b6Fkf5gOUtsnOk0EsrJnA9q_862dzTMNtPrOPplfGoieHU8EbgWFtJ7Kq3OXEcERWgCdCkdisH0ougEwhkTrRKjNfA04XlY4b0w7WJU19iO4o_Pk2PCKr2kVvxLYwEBRfQIcoopnPNS2bseK1jKZbiS6XdeB_WG0QkNNqxMV4ktj5oa1thbgvJIru6tYGXaNRYsdpLT9JlZ2XId8PYLEwoICQ?type=png)

### "UDP deliveries" (no need for Getcho)
* Food -- customers tend to meet delivery drivers. Replacing lost food is usually inexpensive
* "I forgot my keys, can you Uber them to me" and low-value deliveries between peers 

### "TCP deliveries" (great for Getcho)
* Wine -- couriers are required to verify ID. Therefore, customer coordination is essential. Wine return rates without coordination are above 10%.
* Local inventory flows -- showrooms receive $1,000+ merchandise from nearby warehouses several times a day. Getcho helps satellite locations manage stock.
* $200+ payloads like glasses, candles, apparel and specialty items. 
* Gift delivery. We assure the unsuspecting giftee that yes, we do have the right address, and let the gifter know that the delivery was a success. 
* Rentals. Getcho provides identity verification and equipment return scheduling.
* VIP buy-online-pickup-in-store (BOPIS) deliveries. Like delivering cologne to a pitcher at Angel's Stadium.

## The secret sauce

Getcho uses GPS, cargo value, historic deliveries, and communication patterns to derive a delivery risk score. High-risk deliveries receive special attention from Getcho's call center.

Custom AI workflows guide drivers: we transmit building descriptions, access protocol and cargo checks via text and calls.

## Demos

What clients see before sending out a delivery:

https://www.loom.com/share/352e32cdfd6149dca38565c0c673ee7c?sid=bd28d97c-c308-4ad0-92cf-0d9cca26d736

The post delivery report

![](https://getcho.app/images/uploads/getcho-demo.png)

[Here's an example of a lost package we found](https://getcho.app/blog/how-to-save-a-delivery/)

## About us 

Evandro ("Dro") and Jack met on the first day of college. On the last day of college they started a software development firm which they ran for seven years. They built mapping software and an insurance call center system that was acquired.

![](https://pbs.twimg.com/profile_images/579514714419556352/7601EC1W_400x400.jpg)

Mike, Jack's brother, interned with Jack and Dro in college. He joined full-time after a stint at Epic Systems and tripled the company's output and revenue within a year.

![kopticon_austin_3_small](https://github.com/user-attachments/assets/5aac0459-99c3-473e-922f-64f350b16637)

We've been doing local deliveries for the past four years. 

## Ask

Any intros to brick-and-mortar retailers, Shopify sellers and wine merchants
